app/controllers/support_interface/application_forms/application_choices/reinstate_declined_offer_controller.rb

```diff
@@ -24,10 +24,6 @@ def reinstate_offer
         def reinstate_offer_params
           params.require(:support_interface_application_forms_reinstate_declined_offer_form).permit(:status, :audit_comment_ticket, :accept_guidance)
         end
-
-        def redirect_to_application_form_unless_declined_and_reinstate_offer_flag_active
```

Comment:
maybe prudent to leave this redirect in, but for the declined check only?

Reply:
Hmm, both conditions need to be false for that redirect to occur, if we are removing the feature flag, that statement is now redundant because the feature flag part will always be true.

Or am I reading that wrong? 🤔

Reply:
Ah sorry – I mean't leave it there but remove the feature flag part so it's:
```
        def redirect_to_application_form_unless_declined
          redirect_to support_interface_application_form_path(@application_form.id) unless @application_choice.declined?
        end

```

Reply:
but isn't this redirect only happening if the feature flag is disabled?